supabase/migrations/20260816120150_org_member_read_billing_default.sql

```
@@ -0,0 +1,499 @@
+
+-- 1) Default role: every org_member can read plan/usage unless an org removes it.
+INSERT INTO public.role_permissions (role_id, permission_id)
+SELECT r.id, p.id
+FROM public.roles r
+JOIN public.permissions p ON p.key = public.rbac_perm_org_read_billing()
+WHERE r.name = public.rbac_role_org_member()
+ON CONFLICT DO NOTHING;
+
+-- 2) Legacy backfill role for users who previously saw billing via get_orgs_v7
+-- without holding org.read_billing (typically app/channel-only membership).
+INSERT INTO public.roles (name, scope_type, description, priority_rank, is_assignable, created_by)
```

SQLFluff (4.2.2)

[error] Line is too long (98 > 80).

(LT05)